lib/migration-chunks/validation/field.js

```diff
@@ -40,6 +40,10 @@ function fieldHasBeenMoved (validationContext, movementKey) {
   return validationContext.fieldMovements.has(movementKey);
 }
 
+function maybeGetChangedId (validationContext, id) {
+  return validationContext.fieldIdChanges && validationContext.fieldIdChanges[id];
```

Comment:
This will never be undefined, as we create the `Map` (currently: object) before passing in the context